Shared/Samples/Show shapefile metadata/ShowShapefileMetadataView.swift

```diff
@@ -0,0 +1,159 @@
+                }
+                .toolbar {
+                    ToolbarItem(placement: .bottomBar) {
+                        Button("Show Shapefile Metadata") {
+                            showMetadata.toggle()
+                        }
+                        .popover(isPresented: $showMetadata) {
+                            metadataPopover
+                        }
+                    }
+                }
+                .errorAlert(presentingError: $error)
+        }
+    }
+    
+    @ViewBuilder var metadataPopover: some View {
+        NavigationStack {
+            MetadataPanel(model: $model)
+                .navigationTitle("Shapefile Metadata")
+                .navigationBarTitleDisplayMode(.inline)
+                .toolbar {
+                    ToolbarItem(placement: .confirmationAction) {
+                        Button("Done") {
+                            showMetadata = false
+                        }
+                    }
+                }
+        }
+        .presentationDetents([.fraction(0.55)])
+        .frame(idealWidth: 320, idealHeight: 380)
+    }
+}
+
+private extension ShowShapefileMetadataView {
+    @MainActor
+    @Observable
+    class Model {
+        /// Create a map with a topographic basemap.
+        @ObservationIgnored var map = Map(basemapStyle: .arcGISTopographic)
+        
+        /// Declare a FeatureLayer to display the shapefile features on the map.
+        @ObservationIgnored var featureLayer: FeatureLayer!
+        
+        /// Holds metadata information about the shapefile, such as name, description, etc.
+        var shapefileInfo: ShapefileInfo?
+        
+        /// Holds the thumbnail image associated with the shapefile, if available.
+        var thumbnailImage: UIImage?
+        
+        /// Asynchronous function to load the feature layer from the shapefile.
+        func loadShapefile() async throws {
+            let featureTable = ShapefileFeatureTable(fileURL: .auroraShapefile)
+            let layer = FeatureLayer(featureTable: featureTable)
+            
+            try await layer.featureTable?.load()
+            
+            map.addOperationalLayer(layer)
+            featureLayer = layer
+            shapefileInfo = featureTable.info
+            thumbnailImage = featureTable.info?.thumbnail
+        }
+    }
+    
+    struct MetadataPanel: View {
+        /// Binding to the model to reflect changes in the UI.
+        @Binding var model: ShowShapefileMetadataView.Model
+        
+        var body: some View {
+            VStack(alignment: .center, spacing: 16) {
+                if let credits = model.shapefileInfo?.credits {
+                    Text(credits)
+                        .bold()
+                }
+                
+                if let summary = model.shapefileInfo?.summary {
+                    Text(summary)
+                        .font(.caption)
+                        .multilineTextAlignment(.leading)
+                }
```

Comment:
You *could* combine this into one, but I could go either way. Your call.
```suggestion
                if let info = model.shapefileInfo {
                    Text(info.credits)
                        .bold()
                    Text(info.summary)
                        .font(.caption)
                        .multilineTextAlignment(.leading)
                }
```

Reply:
I think it's a good addition, it's in.